examples/discovery-mechanisms/README.md

```diff
@@ -58,34 +54,33 @@ const bootstrapers = [
 Now, once we create and start the node, we can listen for events such as `peer:discovery` and `peer:connect`, these events tell us when we found a peer, independently of the discovery mechanism used and when we actually dialed to that peer.
 
 ```JavaScript
-let node
-
-waterfall([
-  (cb) => PeerInfo.create(cb),
-  (peerInfo, cb) => {
-    peerInfo.multiaddrs.add('/ip4/0.0.0.0/tcp/0')
-    node = new MyBundle({
-      peerInfo
-    })
-    node.start(cb)
+const node = await Libp2p.create({
+  peerInfo,
+  modules: {
+    transport: [ TCP ],
+    streamMuxer: [ Mplex ],
+    connEncryption: [ SECIO ],
+    peerDiscovery: [ Bootstrap ]
+  },
+  config: {
+    peerDiscovery: {
+      bootstrap: {
+        interval: 2000,
+        enabled: true,
+        list: bootstrapers
+      }
+    }
   }
-], (err) => {
-  if (err) { throw err }
-
-  // Emitted when a peer has been found
-  node.on('peer:discovery', (peer) => {
-    console.log('Discovered:', peer.id.toB58String())
-    // Note how we need to dial, even if just to warm up the Connection (by not
-    // picking any protocol) in order to get a full Connection. The Peer Discovery
-    // doesn't make any decisions for you.
-    node.dial(peer, () => {})
-  })
+})
 
-  // Once the dial is complete, this event is emitted.
-  node.on('peer:connect', (peer) => {
-    console.log('Connection established to:', peer.id.toB58String())
-  })
```

Comment:
I thought you suggested to remove it because of being kind of redundant, since should dial is true by default [libp2p/js-libp2p#498/files#r358312128](https://github.com/libp2p/js-libp2p/pull/498/files#r358312128). But yeah, let's put it back, I also think we can double check this